nodecg-io-serial/package.json

```
@@ -0,0 +1,37 @@
+{
+    "name": "nodecg-io-serial",
+    "version": "0.1.0",
+    "description": "Exposes serial deivces to nodecg-io",
+    "homepage": "",
+    "author": {
+        "name": "Stefan Schmelz",
+        "url": "https://github.com/StefanSchmelz"
+    },
+    "scripts": {
+        "build": "tsc -b",
+        "watch": "tsc -b -w",
+        "clean": "tsc -b --clean"
+    },
+    "keywords": [
+        "",
+        "nodecg-bundle"
+    ],
+    "nodecg": {
+        "compatibleRange": "^1.1.1",
+        "bundleDependencies": {
+            "nodecg-io-core": "0.1.0"
+        }
+    },
+    "license": "MIT",
+    "devDependencies": {
+        "@types/node": "^14.6.4",
+        "@types/serialport": "^6.0.1",
```

Is there some reason we need `6.0.1`? I would prefer to have a newer version here also because we're using 9.0.1 of the serialport lib.
```suggestion
        "@types/serialport": "^8.0.1",
```

---

Whoops... This happens if you copy stuff from documentation... Maybe this was giving me these weird type errors.